cmd/gcp-controller-manager/main.go

```diff
@@ -77,6 +77,7 @@ var (
 	hmsSyncNodeURL                     = pflag.String("hms-sync-node-url", "", "URL for reaching the Hosted Master Service SyncNode API.")
 	kubeletReadOnlyCSRApprover         = pflag.Bool("kubelet-read-only-csr-approver", false, "Enable kubelet readonly csr approver or not")
 	autopilotEnabled                   = pflag.Bool("autopilot", false, "Is this a GKE Autopilot cluster.")
+	cleanPreemptibles                  = pflag.Bool("cleanPreemptibles", false, "If true, after preemption, clean pods bound to old node.")
```

Comment:
lgtm, but I would rename this flag because we don't only do this for preemptibles so it's a bit misleading name. How about maybe `clearStalePodsOnNodeRegistration`? Naming is hard :)

Reply:
`ClearStablePodsOnNodeRegistration` sounds good to me. Changed!